client/starwhale/api/_impl/job.py

```diff
@@ -323,3 +379,11 @@ def generate_jobs_yaml(
         ),
         parents=True,
     )
+
+
+class DsInput:
```

Comment:
Are there some limitations for `DsInput` and `ContextInput`? Any object is equal to `DsInput` and `ContextInput`?

Reply:
done